CRM/Report/Form/ActivitySummary.php

```diff
@@ -466,7 +467,7 @@ public function buildQuery($applyLimit = TRUE) {
     $this->groupBy(FALSE);
 
     // build the query to calulate duration sum
-    $sql = "SELECT SUM(activity_civireport.duration) as civicrm_activity_duration_total {$this->_from} {$this->_where} {$this->_groupBy} {$this->_having} {$this->_orderBy} {$this->_limit}";
+    $sql = "SELECT SQL_CALC_FOUND_ROWS SUM(activity_civireport.duration) as civicrm_activity_duration_total {$this->_from} {$this->_where} {$this->_groupBy} {$this->_having} {$this->_orderBy} {$this->_limit}";
```

Comment:
@monishdeb I think you'll need to do a more deliberate count query - check out https://dev.mysql.com/worklog/task/?id=12615 - this is deprecated in mysql 8